src/robot.ts

```diff
@@ -160,7 +169,26 @@ class Robot {
   }
 }
 
-module.exports = (...args) => new Robot(...args)
+export const createRobot = (options: RobotOptions) => new Robot(options)
+
+export interface EventWithEventField {
+  event: string
+}
```

Comment:
Even though we don't use them, should this include all the properties from a webhook?  https://github.com/rvagg/github-webhook-handler/blob/master/github-webhook-handler.js#L92-L99